plugins/woocommerce/src/Admin/Features/PaymentGatewaySuggestions/DefaultPaymentGateways.php

```diff
@@ -390,10 +390,57 @@ public static function get_all() {
 				'image_72x72'         => WC_ADMIN_IMAGES_FOLDER_URL . '/payment_methods/72x72/paypal.png',
 				'plugins'             => array( 'woocommerce-paypal-payments' ),
 				'is_visible'          => array(
-					(object) array(
-						'type'      => 'base_location_country',
-						'value'     => 'IN',
-						'operation' => '!=',
+					self::get_rules_for_countries(
+						array(
+							'US',
+							'CA',
+							'MX',
+							'BR',
+							'AR',
+							'CL',
+							'CO',
+							'EC',
+							'PE',
+							'UY',
+							'VE',
+							'AT',
+							'BE',
+							'BG',
+							'HR',
+							'CH',
+							'CY',
+							'CZ',
+							'DK',
+							'EE',
+							'ES',
+							'FI',
+							'FR',
+							'DE',
+							'GB',
+							'GR',
+							'HU',
+							'IE',
+							'IT',
+							'LV',
+							'LT',
+							'LU',
+							'MT',
+							'NL',
+							'NO',
+							'PL',
+							'PT',
+							'RO',
+							'SK',
+							'SL',
+							'SE',
+							'AU',
+							'NZ',
+							'HK',
+							'JP',
+							'SG',
+							'CN',
+							'ID',
```

Comment:
I think missing `IN`
```suggestion
							'ID',
							'IN',
```

Reply:
This will be addressed in https://github.com/woocommerce/woocommerce/pull/38658/